### PR TITLE
add sleep option to multido to allow time before check

### PR DESCRIPTION
**Pull request by wcpettus**
Purpose of this PR is to add a 'delay_check' option to every entry in a MultiDo, which adds a sleep between the set and the check to fix the power supply output bounce.  I also tried to clarify the error messages to make it easier to tell what fails without diving into the logs by saying which endpoint of a multido failed.

### CODE REVIEW

File: dragonfly/implementations/multido.py [hunk 1]
```diff
@@ -162,7 +168,7 @@ def on_set(self, value):
                 logger.debug('no target_value given: using value ({}) as a target_value to check'.format(value))
                 target_value = value
             if value_get==None:
```
Comment (laroque) [inline dragonfly/implementations/multido.py]: not a line you changed, but you should not check `==None`, should be `if value_get is None`

Comment (wcpettus) [reply dragonfly/implementations/multido.py]: good catch, that shows up a couple times, now fixed

**Comment by laroque** [comment]
did you do an integration test with the device and see it work? if so I'm happy

**Comment by wcpettus** [comment]
I tested it live against the bouncing power supplies and verified that it (a) worked and (b) fixed the known issue.

**Pull request merged**